utils/manifests/utils.go

```diff
@@ -227,6 +227,7 @@ func useDictionary(input string, invert bool) (string, bool) {
 		"caBundle":             "CA Bundle",
 		"mtls":                 "mTLS",
 		"mTLS":                 "mTLS",
+		"fsGroupPolicy":        "fSGroup Policy",
```

Comment:
@Revolyssup @harkiratsm is there a negative ramification caused by changing "additionalProperties" to "Additional Properties"?

Reply:
Yeah it may cause problem related to rjsf schema itself.